.eslintrc.json

```diff
@@ -67,7 +67,8 @@
     "react/no-danger": "off",
     "no-param-reassign": "off",
     "react/require-default-props": "off",
-    "no-useless-constructor": "off"
+    "no-useless-constructor": "off",
+    "sonarjs/no-small-switch": "off"
```

Comment:
Should this be disabled here? I guess you're just disabling it temporarily for `createTemplateFromSession`, where you'd expect the `switch` to grow over time. When the `switch` starts to grow, having this disabled here you'll probably miss it and forget to turn it back on. I'd just disabling it inline with a comment saying to re-enable it when the time comes. Or just get rid of the switch for now!

Reply:
I don't think small switch statements are an issue, which Is why i disabled the rule entirely. The alternative is to use if/else based on the suggestion in https://github.com/SonarSource/eslint-plugin-sonarjs/blob/master/docs/rules/no-small-switch.md.